django/http/request.py

```diff
@@ -555,8 +555,11 @@ def split_domain_port(host):
         return host, ''
     bits = host.rsplit(':', 1)
     if len(bits) == 2:
-        return tuple(bits)
-    return bits[0], ''
+        domain, port = bits
+    else:
+        domain, port = bits[0], ''
```

Comment:
I think this could be simplified to save some lines now the `return`s are gone:
```python
domain, port = bits if len(bits) == 2 else (bits[0], '')
```